src/Microsoft.ML.HalLearners/OlsLinearRegression.cs

```diff
@@ -760,26 +750,6 @@ private protected override void SaveSummary(TextWriter writer, RoleMappedSchema
                 }
             }
         }
-
-        public override void GetFeatureWeights(ref VBuffer<float> weights)
-        {
-            if (_pValues == null)
-            {
-                base.GetFeatureWeights(ref weights);
-                return;
-            }
-
-            var size = _pValues.Length - 1;
-            var editor = VBufferEditor.Create(ref weights, size);
-            for (int i = 0; i < size; i++)
-            {
-                var score = -(float)Math.Log(_pValues[i + 1]);
-                if (score > float.MaxValue)
-                    score = float.MaxValue;
-                editor.Values[i] = score;
-            }
-            weights = editor.Commit();
```

Comment:
It seems that this is lost in the base method?

Reply:
See [https://github.com/dotnet/machinelearning/issues/1850](https://github.com/dotnet/machinelearning/issues/1850)

---
In reply to: [257350745](https://github.com/dotnet/machinelearning/pull/2497#discussion_r257350745) [](ancestors = 257350745)